go/libraries/doltcore/doltdb/doltdb_test.go

```diff
@@ -168,11 +168,14 @@ func TestSystemTableTags(t *testing.T) {
 		assert.Equal(t, queryCatalogMin+4, schema.QueryCatalogDescriptionTag)
 	})
 	t.Run("dolt_schemas tags", func(t *testing.T) {
-		doltSchemasMin := sysTableMin + uint64(4007)
+		doltSchemasMin := sysTableMin + uint64(4012)
```

Comment:
This means that we do a hard fork of the table, instead of just adding columns to it. Any reason that's preferable? You wouldn't be able to diff schemas before and after the cutoff.

Reply:
It’s what we did for v1 to v2, so I’m just mimicking that behavior.